pkg/kn/core/root.go

```diff
@@ -49,9 +49,8 @@ func NewKnCommand(params ...commands.KnParams) *cobra.Command {
 		Short: "Knative client",
 		Long: `Manage your Knative building blocks:
 
-Serving: Manage your services and release new software to them.
-Build: Create builds and keep track of their results.
```

Comment:
Lets remove it from https://github.com/knative/client/blob/master/pkg/kn/core/root.go#L53 , else it will come back.

Reply:
It is already removed, isn't it ? (you are refering to a deleted line)

Reply:
damn!